libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts

```diff
@@ -0,0 +1,74 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
+import { CommonModule } from '@angular/common';
+import { SlideToggleComponent } from './slide-toggle.component';
+import { SlideToggleGroupComponent } from './slide-toggle-group.component';
+import { ToggleData } from './slide-toggle.model';
+import { text, withKnobs, boolean } from '@storybook/addon-knobs';
+
+storiesOf('slide-toggle', module)
+  .addDecorator(
+    moduleMetadata({
+        imports: [CommonModule],
+        declarations: [SlideToggleComponent, SlideToggleGroupComponent],
+ })
+)
+.addDecorator(withKnobs)
+.add(
+    'three different cases about slide-toggle',
+    () => {
+        const data: Array<ToggleData> = [{
+            cod: "1",
+            checked: true,
+            title: "activate color",
+            color: "#10c0c6",
+            uncheckedColor: ""
+        },
+        {
+            cod: "2",
+            checked: true,
+            title: "both colors",
+            color: "#10c0c6",
+            uncheckedColor: "#456888"
+        },
+        {
+        cod: "3",
+        checked: false,
+        title: "UNACTIVATE COLOR",
```

Comment:
Shouldn't the title be 'Inactive'?

Reply:
Or maybe 'unchecked', to keep consistency with component property